Hackney.Shared.Tenure/Domain/TenureInformation.cs

```diff
@@ -31,5 +31,18 @@ public class TenureInformation
         public bool? IsSublet { get; set; }
         public DateTime? SubletEndDate { get; set; }
         public int? VersionNumber { get; set; }
+        public string JigsawRef { get; set; }
+        public string FundingSource { get; set; }
+        public int? NoOfAdults { get; set; }
+        public int? NoOfChildren { get; set; }
+        public bool? HasStorage { get; set; }
+        public bool? RentAccountRequired { get; set; }
+        public string NoRentAccountReason { get; set; }
+        public DateTime? ActualStartOfTenure { get; set; }
+        public DateTime? RentLetterDate { get; set; }
+        public DateTime? RentCardDate { get; set; }
+        public DateTime? MoveOnDate { get; set; }
+        public DateTime? MoveOnAddress { get; set; }
```

Comment:
Please confirm if it's sufficient for this information to be a note on the record (as a comment) or if it needs to be stored as a data property.